packages/algob/src/lib/tx.ts

```diff
@@ -119,6 +119,9 @@ function signTransaction (txn: Transaction, execParams: rtypes.ExecParams): Uint
       return txn.signTxn(execParams.fromAccount.sk);
     }
     case rtypes.SignType.LogicSignature: {
+      const logicSig = execParams.lsig;
+      // Object.assign(logicSig.args, execParams.args ?? []);
+      logicSig.args = execParams.args ?? [];
```

Comment:
how about using `execParams.lsig.args = execParams.args ?? []` ?
this seems more `intuitive`